README.md

```diff
@@ -93,12 +94,12 @@ This repo has an `about.html` file inside of it, as well as a `styles.css` file.
 ## CSS changes:
 
 - Assign a class named `important` to all header tags and make them all underlined and bold.
-- Give the h3 header with the text "Here are some of my Skills!" a unique id `skills-header` and make turn it RebeccaPurple in color.
+- Give the h2 header with the text "Here are some of my Skills!" a unique id `skills-header` and make turn it RebeccaPurple in color.
 - All links that end in .com should be the color green after the link is visited.
-- Give the unordered list of links a class name of `links-list` and make it have a 🌐 symbol as a bullet point.
+- Give the unordered list of links a class name of `links-list` and make it have a 🌐 symbol (`https://github.githubassets.com/images/icons/emoji/unicode/1f310.png`) as a bullet point.
 - In all ordered lists change only the last child element to be bold.
-- Give <a> elements with an href ending ".com" a hover effect of turning red.
-- Give all other <a> elements with an href a hover effect of turning orange.
+- Give <a> elements with an href ending ".com" a focus effect of turning red.
+- Give all other <a> elements with an href a focus effect of turning orange.
```

Comment:
Curious about this. Is this a more correct or accurate way of saying `a hover effect` or do you want the fellows to use the `:focus` pseudo class instead of `:hover`? What is the difference between the two when it comes to `<a>` tags, is it an a11y thing? (I know I should know, sorry)

Reply:
The practical questions for this change are:
- I am not sure we are teaching `:focus` (but I know we teach `:hover`)
- We can't test for `:focus` or `:hover` in cypress anyway

Reply:
Yeah, this is an artifact of me trying to weasel my way out of the `:hover` restrictions by using `:focus` instead... and then hitting a roadblock for them too. I can revert it back to `:hover`.